woocommerce/woocommerce-ios
# Analytics Hub: Create date range section

Summary
==========
Partially fix issue #8144 by introducing the Date Range UI section to the Analytics Hub main view through an AnalyticsTimeRangeCard.

Screen Capture
==========
<img src="https://user-images.githubusercontent.com/5920403/203582871-32ff71e3-3e4c-4f19-8c3d-82319a105089.png" width="322" height="699" />

How to Test
==========
- Launch the app
- Tap the "See More" button from the main dashboard
- See the template hub view

Update release notes:
